index.html

```diff
@@ -35,6 +35,35 @@
   </nav>
 </header>
 
+
+  <div class="mobile-nav d-flex" id="mobile-nav">    
+    <div class="mobile-nav-top d-flex">
+      <img src="svg/mobile-nav-image-top-left.svg" alt="">
+      <div class="cross-img-div"><img src="svg/cross.svg" tabindex="0" id="cancel-img" class="cross-img" alt="click to view menu"></div>
+    </div>
+    <div class="mobile-nav-list d-flex">
```

Comment:
* Kindly fix the mobile menu design as it is different from original fix the width of the mobile menu
[ORIGINAL]
![image](https://user-images.githubusercontent.com/52497246/131392836-5be4a251-5723-471c-b8f6-5956e8178bf0.png)
* [YOURS]
![image](https://user-images.githubusercontent.com/52497246/131393117-7c9e39be-db5f-424d-ba05-187fc36b0a50.png)

Reply:
This issue is not properly resolved!